### ISSUE TITLE
calculating the threshold when applying TSA

### ISSUE THREAD

**Issue by shamanez** [SanghunYun/UDA_pytorch#1]
In this [line](https://github.com/SanghunYun/UDA_pytorch/blob/master/main.py#L88) you have taken the negative exponential of the supervised cross-entropy loss. Can you please explain why you haven't used a softmax output as the confidence of the prediction?

**Comment by SanghunYun** [comment]
Hello shamanez!

The TSA formula is as follows
![image](https://user-images.githubusercontent.com/45596437/68923307-58e97980-07c1-11ea-92bd-fbac7c36eb3d.png)
and sup_loss in torch.exp(-sup_loss)  is as follows
![image](https://user-images.githubusercontent.com/45596437/68923712-4754a180-07c2-11ea-9cef-0366c4e2edf0.png)
So, I calculated prob through torch.exp(-sup_loss)
Of course you can use softamx function like [line89](https://github.com/SanghunYun/UDA_pytorch/blob/1bf65ec253b286976205d646b03efd54dfd6dd10/main.py#L89). But, it is so complex that i used [line88](https://github.com/SanghunYun/UDA_pytorch/blob/1bf65ec253b286976205d646b03efd54dfd6dd10/main.py#L88)
